unity/Assets/HighlightSelectionResponse.cs

```diff
@@ -0,0 +1,7 @@
+﻿using UnityEngine;
+
+internal interface HighlightSelectionResponse
```

Comment:
This is a duplicate of ISelectionResponse.

Reply:
Is it something we need to look at? I think that the HighlightSelectionResponse should be fine?

Reply:
Why do we have duplicate files with different names?

Reply:
I was gonna add a feature but decided not to, kept the code just in case I did want to re-implement it

Reply:
Actually shoot, this was from a previous attempt, this file shouldnt even be here. I think it was from when I was doing stuff in the master branch and then everything got messed up.